app/javascript/controllers/app/search_controller.js

```diff
@@ -24,6 +25,9 @@ export default class extends Controller {
       {{/results}}
     </ul>
   `
+
+    this.throttledAutocomplete = _.throttle(this.autocomplete, 300)
```

Comment:
To me, exposing the idea of throttling requests to the rest of the application is a leaky abstraction. Can we keep the details inside the implementation and keep the public API the same?

Reply:
Good idea! Fixed in 6b85364cb12f5f0249a2798079010fae41f75d9a